File: OCH_Tao/ABC/B/001.py
import math
M = int(input())
km = M/1000
if km < 0.1:
  print("00")
elif 0.1 <= km <= 5:
  if len(str(math.trunc(km*10)))==1:
    print(f"0{math.trunc(km*10)}")
  else:
    print(math.trunc(km*10))
elif 6 <= km <= 30:
  print(50+int(km))
elif 35 <= km <= 70:
  print((int(km)-30)//5+80)
else:
  print("89")

File: OCH_Tao/ABC/B/001.js
const M = parseInt((require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"))[0], 10);
const km = M / 1000;
if (km < 0.1) {
  console.log("00");
} else if (0.1 <= km && km <= 5) {
  if (String(10 * km).length === 1) {
    console.log("0" + Math.trunc(10 * km));
  } else {
    console.log(10 * km);
  }
} else if (6 <= km && km <= 30) {
  console.log(km + 50);
} else if (35 <= km && km <= 70) {
  console.log((km - 30) / 5 + 80);
} else {
  console.log(89);
}